bootstrap/interfaces/secret.go

```diff
@@ -16,4 +16,6 @@ type SecretProvider interface {
 
 	// SecretsLastUpdated returns the last time secrets were updated
 	SecretsLastUpdated() time.Time
+
+	GetAccessToken(tokenType string, serviceKey string) (string, error)
```

Comment:
please add doc/comment for this interface

Reply:
Yep, missed that on.. good catch